### ISSUE TITLE
version up aws-cdk

### ISSUE THREAD

**Issue by osaguild** [osaguild/saitama-premium#29]
message
```
NOTICES

19836   AWS CDK v1 entering maintenance mode soon

        Overview: AWS CDK v1 is entering maintenance mode on June 1, 2022.
                  Migrate to AWS CDK v2 to continue to get the latest features
                  and fixes!

        Affected versions: framework: 1.*, cli: 1.*

        More information at: https://github.com/aws/aws-cdk/issues/19836


If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 19836".
```

**Comment by osaguild** [comment]
- new version is 2.20.0

**Comment by osaguild** [comment]
this issue's pr is included in #42

**Comment by osaguild** [comment]
pr is #45
